### PR TITLE
Update of atracker module

### PR DESCRIPTION
**Pull request by KimberlyH-ibm**

### CODE REVIEW

File: examples/README.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+ # Module atracker
```
Comment (umarali-nagoor) [inline examples/README.md]: Create a directory under examples folder and copy the respective files (main.tf, versions.tf, variables.tf, README etc) inside that

File: metadata_schema.yaml [hunk 2]
```diff
@@ -0,0 +1,558 @@
+# Description : Metadata used to describe a Terraform Module and its contents.
```
Comment (umarali-nagoor) [inline metadata_schema.yaml]: Looks like observability metadata file is being copied here ... please update the file with atracker module info

File: README.md [hunk 3]
```diff
@@ -1 +1,90 @@
-# atracker
\ No newline at end of file
+# IBM Cloud Atracker - Terraform Module
```
Comment (umarali-nagoor) [inline README.md]: Please follow the design guidelines prescribed in (https://github.com/terraform-ibm-modules/getting-started/blob/master/design_guidelines.md) and make the module as Type-1 module

as part of type-1 module, please add root module

File: module/README.md [hunk 4]
```diff
@@ -0,0 +1,102 @@
+# Module atracker
```
Comment (umarali-nagoor) [inline module/README.md]: Create a directory under modules and copy the respective files like main.tf, variables.tf, versions.tf etc which currently under modules directory

File: examples/README.md (hunk 1 above)
Comment (KimberlyH-ibm) [reply examples/README.md]: 11c13c2ff9964f8f18aae14c2db02f7c8b9051fd

File: module/README.md (hunk 4 above)
Comment (KimberlyH-ibm) [reply module/README.md]: https://github.com/terraform-ibm-modules/terraform-ibm-atracker/pull/2#:~:text=update%20module%20directory-,cf33db0,-Merge%20state

File: metadata_schema.yaml (hunk 2 above)
Comment (KimberlyH-ibm) [reply metadata_schema.yaml]: https://github.com/terraform-ibm-modules/terraform-ibm-atracker/pull/2/commits/b3f3fec5c337c68d0dedde3f8b9270f8d84ab493

File: README.md (hunk 3 above)
Comment (KimberlyH-ibm) [reply README.md]: Root module is the atracker or are you referring to the cos and bucket instances?

File: module/README.md (hunk 4 above)
Comment (KimberlyH-ibm) [reply module/README.md]: Are we ready to merge? @umarali-nagoor

File: README.md [hunk 5]
```diff
@@ -1 +1,163 @@
-# atracker
\ No newline at end of file
+# IBM Cloud Activity Tracker with COS - Terraform Module
+<br/>
+
```
Comment (ibmtfprovider) [inline README.md:3]: 1) Add a table with output variables as well.
2) We already have a repo for OBSERVABILITY modules (https://github.com/terraform-ibm-modules/terraform-ibm-observability)
in README, please mention how "atracker" module is different from other observability services like "activity tracker", "sysdig monitoring" and "loogging-logdna"

File: metadata.yaml [hunk 6]
```diff
@@ -0,0 +1,104 @@
...
+        is_required: true
+        description: "Name of the resource group"
+        example: "default"
+        is_sensitive: false
+        related_to: ["data.ibm_resource_group.group"]
+        is_nullable: false
+        is_force_new: false
+        is_provision_controller: false
+        is_count_controller: false
+      - name: "bind_resource_key"
```
Comment (ibmtfprovider) [inline metadata.yaml]: Please capture variables information which are used at root module level in this section.

Sorry, I dont see a variable with name "bind_resource_key" at root level

please cross check the contents of metadata file once

File: README.md [hunk 7]
```diff
@@ -1 +1,163 @@
...
+  cos_instance_id = module.cos.cos_instance_id
+  location        = var.location
+  storage_class   = var.storage_class
+  force_delete    = var.force_delete
+  endpoint_type   = var.endpoint_type
+  kms_key_crn     = var.kms_key_crn
+  expire_rules    = var.expire_rules
+}
+
+module "atracker" {
```
Comment (ibmtfprovider) [inline README.md:56]: Why is it pointing to cos instance module

it should be pointing to root module as you have done in examples/atracker/main.tf

module "atracker" {
  source = "../../"

  resource_group = var.resource_group
  bucket_name    = var.bucket_name
  location       = var.location
  target_crn     = module.cos.cos_instance_id
  api_key        = module.cos.cos_key_credentials.apikey
}

Comment (KimberlyH-ibm) [reply README.md:56]: https://github.com/terraform-ibm-modules/terraform-ibm-atracker/pull/2/commits/091eafa2284351400f813a312e702153ba68f21a

File: README.md (hunk 5 above)
Comment (KimberlyH-ibm) [reply README.md:3]: https://github.com/terraform-ibm-modules/terraform-ibm-atracker/pull/2/commits/e394202b55c21c78d1d6a1fb88b240a78fd3f4ff

**Pull request merged**